Vungle/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Changelog
-  * 6.9.0.0
-    * This version of the adapters has been certified with Vungle 6.9.0 and MoPub SDK 5.14.0.
+  * 6.9.1.0
```

Comment:
minor: this could be same as in gradle "6.9.1.0-early1" to avoid any confusion, when we will be close to GA. then we would update in both places